src/AccessibilityInsights.SharedUx/Settings/ConfigurationManager.cs

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-using Axe.Windows.Desktop.Settings;
 using Axe.Windows.Desktop.UIAutomation;
-using Axe.Windows.RuleSelection;
 using AccessibilityInsights.SetupLibrary;
 using System;
 using System.IO;
 using System.Windows;
+using System.Linq;
+using System.Collections.Generic;
```

Comment:
Do we need System.Linq and System.Collections.Generic? This feels like a remnant of an intermediate change